packages/sw-runtime-caching/src/lib/network-first.js

```diff
@@ -53,6 +59,10 @@ class NetworkFirst extends Handler {
   constructor(input = {}) {
     super(input);
 
+    const cacheableResponse = new CacheableResponse({statuses: [0, 200]});
+    this.requestWrapper.cacheableResponseCheck =
```

Comment:
This line of code  . . . . . wah?

Can this be simplified, abstracted to something more sane, something?